src/utils/hooks/useDataViews.tsx

```diff
@@ -58,14 +55,12 @@ export const useDataViewsTable = ({ tableName, apiQueries }: DataViewTableProps)
   } = useUserState()
 
   useEffect(() => {
-    const queryElements = []
-    if (first) queryElements.push(`first=${first}`)
-    if (offset) queryElements.push(`offset=${offset}`)
-    if (orderBy) queryElements.push(`orderBy=${orderBy}`)
-    if (ascending) queryElements.push(`ascending=${ascending}`)
```

Comment:
nice

Reply:
Yeah, nice not to have to break it apart only to re-assemble it again! ;)